### ISSUE TITLE
How to find the underlying code for MinkowskiConvolution, such as PyTorch's torch.nn.functional.conv2d(weights)？

### ISSUE THREAD

**Issue by XiaoyuAIPro** [NVIDIA/MinkowskiEngine#528]
Dear author, excellent work!
I would like to know whether MinkowskiConvolution can modify the weights of the convolution like PyTorch., e.g. calling nn.F：
`torch.nn.functional.conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=1) [the docs](https://pytorch.org/docs/stable/generated/torch.nn.functional.conv2d.html#torch.nn.functional.conv2d)
This would pass in **custom weights(filters)** for it, if so, **where should I look** for the code and modify it?

I hope you can get back to me, thanks!
If my question wasn't clear, I'll go back and describe it in more detail, and I appreciate your response.

**Comment by yifanpu001** [comment]
same question, looking for the author's reply
